scipy/optimize/_minimize.py

```diff
@@ -40,6 +41,8 @@
                     'l-bfgs-b', 'tnc', 'cobyla', 'slsqp', 'trust-constr',
                     'dogleg', 'trust-ncg', 'trust-exact', 'trust-krylov']
 
+MINIMIZE_METHODS_PY = MINIMIZE_METHODS[:6] + MINIMIZE_METHODS[9:]
```

Comment:
If the order changes we're going to be in trouble. SHould we be explicit?

Reply:
Sure.